trade-finance-manager-ui/api-tests/api-rate-limiting.api-test.js

```diff
@@ -0,0 +1,54 @@
+require('dotenv').config();
+const { createApp } = require('../server/createApp');
+const { createApi } = require('./create-api');
+
+describe('api rate limiting', () => {
+  const rateLimit = process.env.RATE_LIMIT_THRESHOLD;
+
+  let get;
+
+  beforeEach(() => {
+    const app = createApp();
+    get = createApi(app).get;
+  });
+
+  describe('for non-static routes', () => {
+    const sendRequestTimes = (numberOfRequestsToSend) => Promise.allSettled(Array.from({ length: numberOfRequestsToSend }, () => get('/login')));
+
+    it('returns a 429 response if more than RATE_LIMIT_THRESHOLD requests are made from the same IP to the same endpoint in 1 minute', async () => {
```

Comment:
Over 60 seconds assertion please.

Reply:
We've discussed with @abhi-markan and decided to delay adding this test until after resolving the other SR issues this week